bin/ch-build2dir

```diff
@@ -35,7 +35,7 @@ tag=$(basename "$PWD")
 
 set -x
 
-"$CH_BIN"/ch-build -t "$tag" "$context" "$@"
-"$CH_BIN"/ch-docker2tar "$tag" "$dest"
-"$CH_BIN"/ch-tar2dir "${dest}/${tag}.tar.gz" "$dest"
+"${ch_bin:?}"/ch-build -t "$tag" "$context" "$@"
```

Comment:
1. So in principle the `${foo:?}` seems appealing, but we don't use it anywhere else, and in the cases in this PR you're going to get an immediate failure anyway. I think we should just stick with `${foo}` and disable SC2154 in the BATS test.

2. Please change to `"${ch_bin}/ch-build"` (i.e., entire path inside quotation marks) throughout the PR.